code-samples/voice/call-log.js

```diff
@@ -8,31 +7,37 @@ USERNAME     = process.env.RC_USERNAME
 PASSWORD     = process.env.RC_PASSWORD
 EXTENSION    = process.env.RC_EXTENSION
 
-var rcsdk = new RC({
+const rcsdk = new RC({
     server:       SERVER,
     clientId:     CLIENTID,
     clientSecret: CLIENTSECRET
 });
-var platform = rcsdk.platform();
+
+const platform = rcsdk.platform();
+
 platform.login({
     username:  USERNAME,
     password:  PASSWORD,
     extension: EXTENSION
 })
 
-platform.on(platform.events.loginSuccess, () => {
-  read_user_calllog()
+platform.on(platform.events.loginSuccess, read_user_calllog);
+
+platform.on(platform.events.loginError, (e) => {
+    console.error(`User login failed : ${e.message}`)
+    process.exit(1)
```

Comment:
@byrnereese I think we should add a comment here saying "remove this line if you are running the code in node"
Just in case a user decided to copy-paste the code and run it in the browser, `process.exit` will throw runtime error since `process` will not be defined in browser
Thoughts?

Reply:
There is a core assumption that the sample is to be run in node. But that is a good point. Markup the file in a way that you think is ideal. Then let's review that ideal format together and come to agreement. Then we can apply that format to all code samples.

Reply:
@byrnereese I have formatted this file in my recommended way for javascript
- `const/let` for every variable
- camel case for all variable and function names
- semicolon at the end of every statement
- comment to remove/modify lines which are environment specific (node/browser)
- spaces for indentation
- indentation size - 4 spaces

P.S I have added an `.editorconfig` file to define indentation and encoding rules for most of the code editors to pick them are indent the code as per these rules defined in this file

Let me know you thoughts on this